src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -866,4 +942,46 @@ public BuildRule getBinaryBuildRule() {
     }
     return ImmutableSortedSet.of();
   }
+
+
+  @Override
+  public Iterable<? extends NativeLinkable> getNativeLinkableDeps(CxxPlatform cxxPlatform) {
+    return ImmutableSet.of();
+  }
+
+  @Override
+  public Iterable<? extends NativeLinkable> getNativeLinkableExportedDeps(CxxPlatform cxxPlatform) {
+    return ImmutableSet.of();
+  }
+
+  @Override
+  public NativeLinkableInput getNativeLinkableInput(
+      CxxPlatform cxxPlatform, Linker.LinkableDepType type) throws NoSuchBuildTargetException {
+    ImmutableList.Builder<Arg> linkerArgsBuilder = ImmutableList.builder();
+    linkerArgsBuilder.addAll(StringArg.from(
```

Comment:
Since these flags will only be added when a binary is included in a framework bundle, I'm thinking we just do it here, and use the bundle's `platformName` to choose.